plugins/processors/geo/README.md

```diff
@@ -0,0 +1,32 @@
+# Geo Processor Plugin
+
+Use the `geo` processor to add tag with S2 cell ID token of specified [`cell level`][cell levels].
+
+The tag is used in `experimental/geo` Flux package functions.
+
+### Configuration
+
+```toml
+[[processors.geo]]
+  ## The name of the lat and lon fields
+  lat_field = "lat"
+  lon_field = "lon"
+
+  ## New tag to create
+  tag_key = "_ci"
```

Comment:
I don't think we should create any underscore fields in Telegraf by default, these are more or less reserved for use by Flux/InfluxDB.

Reply:
Agreed.  This should the name of the tag for the geohash.  The value of which is the geohash calculated by this processor.  ...and we can default a name for that tag.

For the geo functions in Flux, we should allow the user to specify the geohash tag name in the function call.   This should be reflected here: https://github.com/influxdata/flux/pull/2565

Reply:
Now the tag is also hard coded with underscore in the flux `experimental/geo` pkg so I guess should rename it there as well.

Reply:
Processor is named `s2geo` and tag key `s2cellID`.

Reply:
Great, but can you format it like `s2_cell_id`?

Reply:
@wolffcm Would `s2_cell_id` be ok for you in the geo package?

Reply:
Flux usually prefers camel case, but there are some exceptions, so `s2_cell_id` is fine with me.

Reply:
The rest of the code looks fine, so I'll go ahead and merge this once tag key name is changed.

Reply:
Oops, sorry, I thought this was the Flux PR.

Reply:
Thank you. Tag key renamed to `s2_cell_id`.